resources/assets/pages/ShowGroupType.js

```diff
@@ -63,10 +63,14 @@ const ShowGroupType = () => {
                 className="text-field"
                 onChange={event => setGroupState(event.target.value)}
               >
-                <option value={null}>-- Select State --</option>
+                <option key="null" value={null}>
```

Comment:
Why do you have to set a key and value here? Just curious; don't remember well enough if this is needed and why 🤷‍♂️

Reply:
We don't need the key -- thought we'd need it to avoid the React unique key warning. But we do want to set a value here to clear the `state` filter in the `groups` GraphQL query -- setting it to an empty string instead of null gets this working correctly.